xbmc/powermanagement/PowerManager.h

```diff
@@ -95,6 +96,7 @@ class CPowerManager : public IPowerEventsCallback
   void OnLowBattery() override;
 
   IPowerSyscall *m_instance;
+  CFileItemPtr m_lastItemFile;
```

Comment:
minor, but could you please rename it to `lastFileItem`.

Reply:
sure, but could you describe our coding conventions when to use m_ for member and when we don't use m_?

Seams like I didn't got it ;)

Reply:
the class is called `CFileItem` so it makes sense to reuse it here.